keyboards/creatkeebs/thera/keymaps/default/keymap.c

```diff
@@ -0,0 +1,116 @@
+#include "kb.h"
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance:
```c
// Copyright 2021 Your Name (@yourgithub)
// SPDX-License-Identifier: GPL-2.0-or-later
```

Or 

https://github.com/qmk/qmk_firmware/blob/6b299fab6e6a3143c061b9ea005253361fcd524b/data/templates/base/keyboard.c#L1-L15

```suggestion
#include QMK_KEYBOARD_H
```